scripts/update-vm.sh

```diff
@@ -24,11 +24,7 @@ setup_chefdk() {
     echo "ChefDK $CHEFDK_VERSION already installed"
   else
     step "Downloading and installing ChefDK $CHEFDK_VERSION"
-    mkdir -p $DOWNLOAD_DIR
-    local CHEFDK_DEB=chefdk_$CHEFDK_VERSION-1_amd64.deb
-    local CHEFDK_URL=https://packages.chef.io/files/stable/chefdk/$CHEFDK_VERSION/ubuntu/16.04/$CHEFDK_DEB
-    [[ -f $DOWNLOAD_DIR/$CHEFDK_DEB ]] || sudo wget --no-verbose -O $DOWNLOAD_DIR/$CHEFDK_DEB $CHEFDK_URL
-    sudo dpkg -i $DOWNLOAD_DIR/$CHEFDK_DEB
+    curl -L https://omnitruck.chef.io/install.sh | sudo bash -s -- -P chefdk -c stable -v $CHEFDK_VERSION -d /tmp/vagrant-cache
```

Comment:
FYI - the reason I had the more complicated version earlier was that I wanted to have this script work even if I was completely offline-

I'm fine with the single line curl / bash pipe too (it's the more idiomatic and bulletproof way to install chef anyway), so let's keep it!